src/signature_pad.js

```diff
@@ -404,7 +406,7 @@ SignaturePad.prototype._toSVG = function () {
 
         path.setAttribute('d', attr);
         path.setAttribute('stroke-width', (widths.end * 2.25).toFixed(3));
-        path.setAttribute('stroke', this.penColor);
+        path.setAttribute('stroke', color);
```

Comment:
So this works:

1. Create a design with multiple colors.
2. Export to SVG saves design with correct color combinations.
3. Import from the same SVG shows color combinations correctly.

What doesn't work:

At that point you cannot export to SVG again using 2.1.1 even without the proposed change since the SVG export feature relies on recording the strokes to the this._data array.

The only solution I see would be to repopulate the this._data array by reinterpreting the imported SVG data instead of relying on the canvas itself to draw the imported data.